### PR TITLE
Improve clarity around SPM installation instructions

### LINKED ISSUE #46: Difficulty following spm setup instructions
It took me several tries and some research to figure out how to follow the [spm setup instructions](https://github.com/rogerluan/arkana#adding-a-local-swift-package-to-your-xcode-project)

Step 1 may be a bit confusing to people as the default hierarchy is ArkanaKeys > ArkanaKeys. The instructions say to select the ArkanaKeys folder, without being specific for which one (it is of course the inner one).

Step 3 was hard to follow because there appears to be an issue that your local package you added above does not show up in the list to choose from unless you quit and reload xcode.

### PR DESCRIPTION
**Pull request by rogerluan**
Resolves #46

### CODE REVIEW

**Comment by rogerluan** [comment]
Hey @dalewking, would this make the documentation easier to follow? 😊

**Comment by rogerluan** [comment]
Friendly ping @dalewking 😄

**Review by dalewking — APPROVED** [review]
